### ISSUE TITLE
2.1 pro - Multi Site Manager MSM - button not displaying correctly

### ISSUE THREAD

**Issue by garethshaw** [pyrocms/pyrocms#1415]
I get this after a fresh install! Guess the Language translations are missing as it's showing the Alt text.

Shared Modules
Upload
Name    Description     Version  
API Management  Set up a RESTful API with API Keys and out in JSON, XML, CSV, etc.  1.0.0   http://pyrocms210.mknmedia.com/sites/addons/disable/camsystems/module/api    http://pyrocms210.mknmedia.com/sites/addons/uninstall/camsystems/module/api/1 Delete
